### ISSUE TITLE
[improve][doc] flags (parameters) of `pulsar-daemon start` are missing in docs

### ISSUE THREAD

**Issue by Anonymitaet** [apache/pulsar#19512]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### TL;DR

Flags (parameters) of `pulsar-daemon start` are missing in the [pulsar-daemon reference doc](https://pulsar.apache.org/reference/#/next/pulsar-daemon/pulsar-daemon).

### Details

As discussed with @horizonzy, to get started with Pulsar more quickly, users can start Pulsar with some commands, arguments, and parameters.

For example, `bin/pulsar-daemon start standalone -nfw -nss` can start Pulsar without function workers and stream storage in the background, which is **faster** and better suits newbies' needs.

Currently, [pulsar-daemon docs](https://pulsar.apache.org/reference/#/next/pulsar-daemon/pulsar-daemon) tells users that they can get more info (commands and arguments) using `pulsar-daemon`(**this solution is acceptable and makes sense**). However,  `pulsar-daemon` **does not provide flags (parameters)**. 

<img width="1960" alt="image" src="https://user-images.githubusercontent.com/50226895/218656216-a9e91a61-1de6-4f94-b6e8-fd3611af2564.png">

Veteran developers can find these flags (parameters) in the [code file - PulsarStandalone.java, L214 - L268](https://github.com/apache/pulsar/blob/70c4003c5c5ef6d911f639d946c235721b6d0e67/pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java#L214), while some users does not know or it's inconvenient for them to get those info in the shortest time.

### Solution

Hi @horizonzy, 

Double-check: are the`pulsar-daemon start standalone flags` exactly same to [`pulsar standalone flags`](https://pulsar.apache.org/reference/#/next/pulsar/standalone)? They all use the parameters in [PulsarStandalone.java, L214 - L268](https://github.com/apache/pulsar/blob/70c4003c5c5ef6d911f639d946c235721b6d0e67/pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java#L214)?

If so, how about adding a tip like below?

<img width="976" alt="image" src="https://user-images.githubusercontent.com/50226895/218658530-c66fb574-13f3-4200-a02b-ae0a2306ee46.png">

> TIP: You can use flags to perform more tasks. For example, `pulsar-daemon start standalone -nfw -nss` can start Pulsar faster in background (without function workers and stream storage). `pulsar-daemon start standalone` flags are same as `pulsar standalone` flags. For the complete flag list, see flags of the [`pulsar standalone` command](https://pulsar.apache.org/reference).

🔹🔹🔹

cc @hangc0276 @momo-jun @DaveDuggins @D-2-Ed

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by horizonzy** [comment]
> Hi @horizonzy,
>Double-check: are thepulsar-daemon start standalone flags exactly same to [pulsar standalone flags](https://pulsar.apache.org/reference/#/next/pulsar/standalone)? They all use the parameters in [PulsarStandalone.java, L214 - L268](https://github.com/apache/pulsar/blob/70c4003c5c5ef6d911f639d946c235721b6d0e67/pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java#L214)?

Yes.

>TIP: You can use flags to perform more tasks. For example, pulsar-daemon start standalone -nfw -nss can start Pulsar faster in background (without function workers and stream storage). pulsar-daemon start standalone flags are same as pulsar standalone flags. For the complete flag list, see flags of the [pulsar standalone command](https://pulsar.apache.org/reference).

The link is not match the content.

**Comment by Anonymitaet** [comment]
@horizonzy thanks for your confirmation!

> The` link is not match the content.

When providing links to the Pulsar Reference site, we suggest using the general link (https://pulsar.apache.org/reference) instead of specific links (e.g., https://pulsar.apache.org/reference/#/next/pulsar/standalone) for all versioned docs to reduce maintenance costs. Readers can choose their desired pages after they are navigated to the home page. We apply this strategy in our daily work to keep consistent, e.g., https://github.com/apache/pulsar/issues/19491#issuecomment-1428991305.

**Comment by momo-jun** [comment]
@Anonymitaet for a specific command like this one, I think we can leave it in plain text and add another sentence to link the reference site, for example, "See the reference docs for more details.". 
Making it a link may give users a sense that the redirection exactly points to this command reference information, while consequently making them disappointed.

**Comment by Anonymitaet** [comment]
> @Anonymitaet for a specific command like this one, I think we can leave it in plain text and add another sentence to link the reference site, for example, "See the reference docs for more details.". Making it a link may give users a sense that the redirection exactly points to this command reference information, while consequently making them disappointed.

Hi @momo-jun thanks! I think my proposed change has already matched your thoughts?
<img width="872" alt="image" src="https://user-images.githubusercontent.com/50226895/219231160-ff14a9e8-e348-44ba-9fbf-7db30eb7e3d3.png">

**Comment by momo-jun** [comment]
> Hi @momo-jun thanks! I think my proposed change has already matched your thoughts? 

The addition is good. My suggestion is to replace the display text of the link with "reference docs". Because it redirects the users to a holistic doc site rather than a specific command.
